setup.py

```diff
@@ -48,16 +48,15 @@
                 'testcontainers>=2.3',
                 'future==0.17.1',
                 'coveralls>=1.7.0',
-                'dali-dataset',
             ],
             'docs': [
-                'DALI-dataset>=1.0.0',
+                'dali-dataset==1.1',
```

Comment:
To double check - In line 31, is `import DALI` still correct with the new version of `dali-dataset`?

Reply:
yes, i just double-checked